packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -87,6 +86,13 @@ import {
  * @import {AnyType, MsgDepositForBurnType, MsgUndelegateResponseType} from '../utils/codecs.js';
  */
 
+const TOMBSTONED_WATCHERS = /** @type {const} */ ([
```

Comment:
good name. please comment too on what this is for and when/how it can be changed